src/services/AppUserService.ts

```diff
@@ -0,0 +1,18 @@
+import { AppUser } from '@Entities';
+import { injectable } from 'tsyringe';
+import { Any, getRepository } from 'typeorm';
+import { AppUserServiceInterface } from './interfaces/AppUserServiceInterface';
+import { Repository } from 'typeorm';
+
+@injectable()
+export class AppUserService implements AppUserServiceInterface {
+  private appUserRepository: Repository<AppUser>;
+
+  constructor() {
+    this.appUserRepository = getRepository(AppUser);
+  }
+
+  getMultipleAppUsers(ids: number[]): Promise<AppUser[]> {
+    return this.appUserRepository.find({ id: Any(ids) });
```

Comment:
Just curious on how this line works

Reply:
https://orkhan.gitbook.io/typeorm/docs/find-options - maps to ANY in sql
python equivalent is sth like appUserTable.filter(n -> n in ids)